tools/filters/gff/gff_filter_by_attribute.py

```diff
@@ -103,25 +113,24 @@ def get_value(name, a_type, values_dict):
     total_lines += 1
     line = line.rstrip( '\\r\\n' )
     if not line or line.startswith( '#' ):
-        skipped_lines += 1
-        if not invalid_line:
-            first_invalid_line = i + 1
-            invalid_line = line
+        # Ignore blank lines or comments
         continue
     try:
         # Place attribute values into variables with attribute
         # name; type casting is done as well.
         elems = line.split( '\t' )
         attribute_values = {}
         for name_value_pair in elems[8].split(";"):
-            pair = name_value_pair.strip().split(" ")
-            if pair == '':
+            # Split on first equals (GFF3) or space (legacy)
+            name_value_pair = name_value_pair.strip()
+            i = name_value_pair.replace(" ", "=").find("=")
```

Comment:
According to http://gmod.org/wiki/GFF3, space are allowed in the `attributes` column, so this would not work on some GFF3 files.